contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@gierschv there are certainly other ways to spin up a ceph cluster (e.g. ceph-deploy). the micro-osd.sh script worked well in the travis-ci environment where there were issues with ceph-deploy that i never worked out.